docs/en/custom.rst

```diff
@@ -100,7 +100,8 @@ you can simplify this to:
     }
 
 Alternatively, you can annotate your annotation class with
-``@NamedArgumentConstructor`` in case you cannot use the marker interface.
+``@NamedArgumentConstructor`` in case you cannot use the marker interface. 
+In this case, the first argument of the constructor will be considered as the default property. 
```

Comment:
The way this is written makes me think that this works only when using the `@NamedArgumentConstructor` annotation but not when using the marker interface, as you added that in the paragraph about the alternative. Is it actually true ?

Reply:
It is true. It is the intended behaviour as we didn't want to introduce breaking change with current implementation of the interface.
See https://github.com/doctrine/annotations/issues/396